src/app/components/tasks/tasks.component.ts

```diff
@@ -0,0 +1,115 @@
+// angular
+import { Component, OnInit, OnDestroy } from '@angular/core';
+
+// services
+import {TaskService} from '../../services/task.service';
+
+// entities
+import { Task } from '../../entities/task';
+
+@Component({
+  selector: 'app-tasks',
+  templateUrl: './tasks.component.html',
+  styleUrls: ['./tasks.component.css']
+})
+export class TasksComponent implements OnInit {
+
+  // property ( get tasks from task service ) 
+  get tasks() {
+    return this.srv.tasks;
+  }
+
+  // filter for show/hide tasks
+  filter: string = 'all';
+
+  constructor(
+    private srv: TaskService
```

Comment:
What is srv? Where did you spend bunch of time that you get from shortcutting service to srv?